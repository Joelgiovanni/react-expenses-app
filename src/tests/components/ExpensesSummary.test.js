import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpenesSummary';

test('Should render expenses summary with 1 expense', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={200}/>);
  expect(wrapper).toMatchSnapshot();
})

test('Should render expenses summary with multiple expenses', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={4} expensesTotal={4000000000}/>);
  expect(wrapper).toMatchSnapshot();
})