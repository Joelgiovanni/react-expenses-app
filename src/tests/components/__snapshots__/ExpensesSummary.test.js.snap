// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render expenses summary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
    . Total of: 
    $2.00
  </h1>
</div>
`;

exports[`Should render expenses summary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    4
     
    expenses
    . Total of: 
    $40,000,000.00
  </h1>
</div>
`;
